packages/bee-q/src/components/breadcrumb-item/bq-breadcrumb-item.tsx

```diff
@@ -0,0 +1,133 @@
+import { h, Component, Prop, Element, Event, EventEmitter } from '@stencil/core';
+
+import { isDefined } from '../../shared/utils';
+
+@Component({
+  tag: 'bq-breadcrumb-item',
+  styleUrl: './scss/bq-breadcrumb-item.scss',
+  shadow: true,
+})
+export class BqBreadcrumbItem {
+  // Own Properties
+  // ====================
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLBqBreadcrumbItemElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  /** If true, the item is the last element inside breadcrumb */
+  @Prop() hideSeparatorIcon: boolean = false;
+
+  /** The icon name used as separator. Default is `/`. */
+  @Prop() separatorIcon!: string;
```

Comment:
Shouldn't the separator icon be at the parent level (`bq-breadcrumb`)? Otherwise, a consumer could end up with different separators, right? Also, it will be easy (in case the consumer would like to use a custom one) to set it once and not on each breadcrumb item.

Reply:
The separator icon is on the parent level, and this variable within the item component it can only be set from the `bq-breadcrumb`. The variable has 2 states:
- undefined => use `/` as separator;
- icon name => display `bq-icon` and `separatorIcon` holds the name set by user